shopinvader_customer_multi_user/models/res_partner.py

```diff
@@ -29,6 +29,18 @@ class ResPartner(models.Model):
         compute="_compute_has_invader_user",
         help="At least one backend has an invader user for this partner.",
     )
+    invader_address_share_policy = fields.Selection(
+        [
+            ("public", "Public"),
+            ("private", "Private"),
+        ],
+        string="Shop Address Share Policy",
+        help="In a multi user environment, controls who can view this address\n\n"
+        "* `Public`: Shared among all company's users.\n"
+        "* `Private`: Only the user who created it and company's admin users.\n",
+        default="public",
+        required=True,
```

Comment:
it's not required IMO

Reply:
Hmm not sure. If it's not required then we have to deal with null values, which wouldn't make much sense IMO.
Not using this feature means `default="public"` IMO, no?

Reply:
From my POV this field is required only when tied to shopinvader. For the rest of the ecosystem it should not.
I don't see any reason to make it required, especially because we have a default.
In the endpoint validator you can pass a default if you want to make it always valued.